src/api/wayfire/view_access_interface.hpp

```diff
@@ -0,0 +1,27 @@
+#ifndef VIEW_ACCESS_INTERFACE_HPP
+#define VIEW_ACCESS_INTERFACE_HPP
+
+#include "wayfire/condition/access_interface.hpp"
+#include "wayfire/view.hpp"
+#include <string>
+#include <tuple>
+
+namespace wf
+{
+
+class view_access_interface_t : public access_interface_t
```

Comment:
Okay. I tend to doc things late in the development because there is less chance of the code being changed and the docs forgotten and then going out of sync with the code. I'll doc it.